LayoutKitTests/TextViewLayoutTests.swift

```diff
@@ -0,0 +1,163 @@
+        let attributedString = NSAttributedString(
+            string: textString,
+            attributes: [NSFontAttributeName: UIFont.helvetica(size: 15)])
+        let attributedText = Text.attributed(attributedString)
+
+        let textView = UITextView()
+        textView.attributedText = attributedString
+        textView.updateAttributes()
+
+        let layout = TextViewLayout(text: attributedText)
+        let layoutView = layout.arrangement()
+
+        XCTAssertEqual(layoutView.frame.size, textView.intrinsicContentSize)
+    }
+
+    func testMutipleLinesWithMutipleAttributedStrings() {
+        let textString = "Hello World\nHello World\nHello World\nHello World\nHello World"
+        let attributedString1 = NSMutableAttributedString(
+            string: textString,
+            attributes: [NSFontAttributeName: UIFont.helvetica(size: 15)])
+        let attributedString2 = NSMutableAttributedString(
+            string: textString,
+            attributes: [NSFontAttributeName: UIFont.helvetica(size: 12)])
+        attributedString1.append(attributedString2)
+        let attributedText = Text.attributed(attributedString1)
+
+        let textView = UITextView()
+        textView.attributedText = attributedString1
+        textView.updateAttributes()
+
+        let layout = TextViewLayout(text: attributedText)
+        let layoutView = layout.arrangement()
+
+        XCTAssertEqual(layoutView.frame.size, textView.intrinsicContentSize)
+    }
+
+    func testLineFragmentPadding() {
+        let textString = "Hello World\nHello World\nHello World\nHello World\nHello World"
+        let text = Text.unattributed(textString)
+        let lineFragmentPadding: CGFloat = 30.0
+
+        let textView = UITextView()
+        textView.text = textString
+        textView.font = defaultTextViewLayoutFont
+        textView.updateAttributes(lineFragmentPadding: lineFragmentPadding)
+
+        let layout = TextViewLayout(text: text, lineFragmentPadding: lineFragmentPadding)
+        let layoutView = layout.arrangement()
+
+        XCTAssertEqual(layoutView.frame.size, textView.intrinsicContentSize)
+    }
+
+    func testTextContainerInset() {
+        let textString = "Hello World\nHello World\nHello World\nHello World\nHello World"
+        let text = Text.unattributed(textString)
+        let textContainerInset = UIEdgeInsets(top: 2, left: 3, bottom: 4, right: 5)
+
+        let textView = UITextView()
+        textView.text = textString
+        textView.font = defaultTextViewLayoutFont
+        textView.updateAttributes(textContainerInset: textContainerInset)
+
+        let layout = TextViewLayout(text: text, textContainerInset: textContainerInset)
+        let layoutView = layout.arrangement()
+
+        XCTAssertEqual(layoutView.frame.size, textView.intrinsicContentSize)
+    }
+
+}
+
+// MARK: - helper extension
+
+private extension UITextView {
+
+    func updateAttributes(lineFragmentPadding: CGFloat = 0,
```

Comment:
instead of having this be a method you have to call, I would just create a convenience initializer like LabelLayoutTests.